sockeye/train.py

```diff
@@ -607,9 +625,8 @@ def create_optimizer_config(args: argparse.Namespace) -> OptimizerConfig:
     config.set_lr_scheduler(lr_sched)
     logger.info("Optimizer: %s | kvstore=%s | params=%s | initializer=%s",
                 config.name, config.kvstore, config.params, config.initializer)
-    if args.update_interval > 1:
-        logger.info("Gradient accumulation over %d batches. Effective batch size: %d",
-                    args.update_interval, effective_batch_size)
+    logger.info("Gradient accumulation over %d batch(es) by %d worker(s). Effective batch size: %d",
+                args.update_interval, num_workers, effective_batch_size)
     return config
```

Comment:
You all should consider requiring Python 3.7+ for Sockeye 2, just for the f-strings.

Reply:
Yes they are nice, but I think there is still value in supporting Python 3.6